src/Forms/ExportPngXml.cs

```diff
@@ -3611,6 +3606,7 @@ internal void InitializeFromVobSubOcr(Subtitle subtitle, SubtitleFormat format,
 
         private void subtitleListView1_SelectedIndexChanged(object sender, EventArgs e)
         {
+            if (_isLoading) return;

```

Comment:
Will prevent calling this method every time we set combobox to a value when initializing.